packages/cfd/src/Containers/cfd-password-modal/cfd-password-form/cfd-password-form.tsx

```diff
@@ -0,0 +1,271 @@
+import React from 'react';
+import { Formik, FormikErrors } from 'formik';
+import { getLegalEntityName, getErrorMessages, getCFDPlatformLabel, CFD_PLATFORMS, CFD_text } from '@deriv/shared';
+import { Text, FormSubmitButton, PasswordMeter, PasswordInput } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import {
+    getDxCompanies,
+    getFormattedJurisdictionCode,
+    TDxCompanies,
+} from '../../../Stores/Modules/CFD/Helpers/cfd-config';
+import { TCFDPasswordFormReusedProps, TCFDPasswordFormValues, TOnSubmitPassword } from '../../props.types';
+import { ACCOUNT_CATEGORY, PASSWORD_ERRORS } from '../../../Constants/cfd-password-modal-constants';
+import PasswordStep from './password-step';
+import CFDPasswordError from './cfd-password-error';
+import { observer, useStore } from '@deriv/stores';
+
+type TCFDPasswordFormProps = TCFDPasswordFormReusedProps & {
+    account_title: string;
+    account_type: {
+        category?: string;
+        type?: string;
+    };
+    closeModal: () => void;
+    error_type?: string;
+    form_error?: string;
+    has_mt5_account: boolean;
+    is_bvi: boolean;
+    is_dxtrade_allowed: boolean;
+    is_real_financial_stp: boolean;
+    jurisdiction_selected_shortcode: string;
+    onCancel: () => void;
+    onForgotPassword: () => void;
+    should_set_trading_password: boolean;
+    show_eu_related_content: boolean;
+    submitPassword: TOnSubmitPassword;
+};
+
+const CFDPasswordForm = observer(
+    ({
+        account_title,
+        account_type,
+        closeModal,
+        error_message,
+        error_type,
+        form_error,
+        has_mt5_account,
+        is_real_financial_stp,
+        jurisdiction_selected_shortcode,
+        onCancel,
+        onForgotPassword,
+        platform,
+        should_set_trading_password,
+        show_eu_related_content,
+        submitPassword,
+        validatePassword,
+    }: TCFDPasswordFormProps) => {
+        const { ui } = useStore();
+        const { is_mobile } = ui;
+
+        const button_label = React.useMemo(() => {
+            if (error_type === PASSWORD_ERRORS.RESET) {
+                return localize('Try later');
+            }
+            return localize('Add account');
+        }, [error_type]);
+
+        const getCancelButtonLabel = ({
+            should_set_trading_password,
+            error_type,
+        }: Pick<TCFDPasswordFormProps, 'should_set_trading_password' | 'error_type'>) => {
+            if (should_set_trading_password && error_type !== PASSWORD_ERRORS.RESET) {
+                return is_mobile ? localize('Cancel') : null;
```

Comment:
use <Localize /> instead of localize
```suggestion
                return is_mobile ? <Localize  i18n_default_text='Cancel'/> : null;
```